client/components/app.jsx

```diff
@@ -24,6 +30,23 @@ class App extends React.Component {
     });
   }
 
+  getCartItems() {
+    fetch('/api/cart', { method: 'GET' })
+      .then(res => res.json)
+      .then(data => this.setState({ cart: data }));
+  }
+
+  addToCart(product) {
+    // const productItem = { product: '' };
+    fetch('/api/cart', {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify(product)
+    })
+      .then(res => res.json())
+      .then(() => this.setState({ cart: product }));
```

Comment:
Here you need to capture the newly added product with a parameter here then it needs to be added to the cart array in state